### PR TITLE
initial cut at demographic data

### PR DESCRIPTION
**Pull request by bjmckenz**
I'd appreciate a review. It's not bad, but I think I need feedback.

* All people (admins, students, profs) have representative data
* Data can be edited
* summaries of profs and students are on the new page

Looking for code review and usability (and naming of stuff)

### CODE REVIEW

**Review by oceancypress — COMMENTED** [review]
seems code looks good, about to test

File: student-info-system/sis/models.py [hunk 1]
```diff
@@ -56,6 +56,119 @@ def has_professor(self):
...
+        ('Hispanic', 'Hispanic'),
+        ('Black', 'Black'),
+        ('American Indian/Alaska Native', 'American Indian/Alaska Native'),
+        ('Decline to State', 'Decline to State'),
+    )
+    demo_race = models.CharField(max_length=40, blank=True, choices=RACE)
+    GENDER = (
+        ('Male', 'Male'),
+        ('Female', 'Female'),
+        ('Trans', 'Trans'),
```
Comment (oceancypress) [inline student-info-system/sis/models.py:94]: Can we make this non-binary? it would encompass more people

**Review by oceancypress — CHANGES_REQUESTED** [review]

**Comment by oceancypress** [comment]
Drop down for household income in the edit pages is not selectable. Other than that looks great

File: student-info-system/sis/models.py (hunk 1 above)
Comment (bjmckenz) [reply student-info-system/sis/models.py:94]: added

**Pull request merged**